tests/Optimizer/Transformer/OptimizeHeroImagesTest.php

```diff
@@ -341,6 +341,28 @@ public function dataTransform()
                     . '</amp-story-player>'
                 ),
             ],
+
+            'preserves fetchpriority attribute from noscript fallback img' => [
+                $input(
+                    '<amp-img width="500" height="400" src="/img1.png"><noscript><img src="/img1.png" width="500" height="400" fetchpriority="high"></noscript></amp-img>'
+                ),
+                $output(
+                    '<amp-img data-hero width="500" height="400" src="/img1.png" i-amphtml-ssr><img class="i-amphtml-fill-content i-amphtml-replaced-content" decoding="async" fetchpriority="high" src="/img1.png"></amp-img>'
+                ),
+            ],
+
+            'automatically ssr img element with high fetchpriority attribute from noscript fallback img' => [
+                $input(
+                    '<amp-img width="500" height="400" src="/img1.png"><noscript><img src="/img1.png" width="500" height="400" fetchpriority="high"></noscript></amp-img>'
+                    .  '<amp-img width="500" height="400" src="/img2.png"><noscript><img src="/img2.png" width="500" height="400"></noscript></amp-img>'
+                    .  '<amp-img width="500" height="400" src="/img3.png"><noscript><img src="/img3.png" width="500" height="400" fetchpriority="high"></noscript></amp-img>'
```

Comment:
For the second image, how about adding `data-hero` to it to see what happens when preceded by a `fetchpriority=high` image, and then the third image can have neither `data-hero` nor `fetchpriority` (as really the source markup shouldn't have multiple `fetchpriority=high`.